client/components/popover/README.md

```diff
@@ -1,40 +1,47 @@
 Popover
 =======
 
-`Popover` is a react component that can be used to show any content in a popover.
+`Popover` is a react component that can be used to show any content in a
```

Comment:
Just curious -- what happened w/ the formatting of this Readme file? I'm not 💯, but don't think we impose a max line length for .md files.

Reply:
When I made my change to the file I saw it adhered to a hard line length in most parts, but not in a bunch of others. This seemed like an opportunity to pick up some (organically grown) trash. In hindsight it would've been better to isolate this to its own PR.